### ISSUE TITLE
add support for FTS tables

### ISSUE THREAD

**Issue by yigit** [greenrobot/greenDAO#52]
android ships with FTS3 support (FTS4 on honeycomb +)
it would be nice to support FTS tables to provide better full text search.

it might be also good to support adding SQL triggers automatically from entity tables.

details:
http://www.sqlite.org/fts3.html

**Comment by GaminTsai** [comment]
does it support FTS3 or fts4 now ?

**Comment by skrzyneckik** [comment]
@greenrobot does anybody consider it is even possible to fit it into GreenDAO? [I've seen](http://stackoverflow.com/questions/10904558/fts3-searches-in-ormlite) that ORMLite recommends using raw queries. Maybe it should be also recommended for GreenDAO/solution for this issue?

**Comment by TimVelo** [comment]
I think it would make a really useful feature if we could do FTS tables using this awesome library! 👍   

@skrzyneckik I'm guessing you'd have to write the table as ad-hoc (using FTS commands), and not let DAO generate the non-FTS table.

**Comment by skrzyneckik** [comment]
@TimVelo I agree it would be useful feature, but this issues is already 4 years old, so I guess solution isn't trivial. I don't want to speculate because I don't know GreenDAO architecture, but I would like to hear an opinion from somebody involved in this project.
